numpy/distutils/fcompiler/__init__.py

```diff
@@ -480,6 +483,14 @@ def customize(self, dist = None):
             freeflags = self.flag_vars.free
         # XXX Assuming that free format is default for f90 compiler.
         fix = self.command_vars.compiler_fix
+        # NOTE: this and similar examples are probably just
+        # exluding --coverage flag when F90 = gfortran --coverage
+        # instead of putting that flag somewhere more appropriate
+        # this and similar examples where a Fortran compiler
+        # environment variable has been customized by CI or a user
+        # should perhaps eventually be more throughly tested and more
+        # robustly handled
```

Comment:
I suppose this comment may be out of date after Eric's `[0]` removal suggestion, though not confident compiler flag handling is perfect at the moment.